features/user/reset_password.feature

```diff
@@ -0,0 +1,17 @@
+@reset-password @notify
+Feature: Reset user password
+
+Scenario: Request password reset
+  Given I have a buyer user
+  And I am on the /user/reset-password page
+  When I enter that user.emailAddress in the 'Email address' field
+  And I click 'Send reset email' button
+  Then I see a success banner message containing 'send a link to reset the password'
```

Comment:
Hmm, it feels like the content might have been better as `If the email address you've entered belongs to a Digital Marketplace account, we've sent a link to reset the password.` ... But I guess that's a different matter.